src/dispatchers/touchDispatcher.ts

```diff
@@ -0,0 +1,138 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export type TouchCallback = (p: Point) => any;
+
+  export class Touch extends AbstractDispatcher {
+    private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Touch";
+    private translator: _Util.ClientToSVGTranslator;
```

Comment:
Since they're independent, let's solve that issue in another PR.  It'll make this PR's lifetime shorter